extension/intents/music/music.toml

```diff
@@ -156,3 +156,15 @@ phrase = "What is playing"
 [[music.showTitle.example]]
 phrase = "What's playing?"
 test = true
+
+[music.playAlbum]
+description = "Play albums or playlists from a music service"
+match = """
+  (play | open) (album | playlist) [query] (on | in) [service:musicServiceName]
+  (play | open) [query] (album | playlist) (on | in) [service:musicServiceName]
```

Comment:
Album and playlist should be separate. They could use a parameter (e.g., `(play | open) playlist [query] (on | in) [service:musicServiceName] [type=playlist]`), or there could be two separate intents. Either seems fine to me.

Reply:
@ianb Can you tell a bit more about how should album and playlist be done. What are the things that should be done?

Reply:
Can you tell a bit more about the differences between album and playlist? What should be triggered when the user asks for a playlist?

Reply:
But it looks like you've started with one of these. I suppose you could also play the first result from one of these sections without navigating to the specific search result.

Reply:
@ianb Added support for playlist playing.